src/App.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
+import React from "react";
+
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+const toDoData = [
+  {
+    task: "Organize Garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "Bake Cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      toDos: toDoData,
+      id:Date.now(),
+      inputText:''
+    };
+  }
+  handler = event => {
+    this.setState({
+      inputText: event.target.value
+    });
+    console.log(event.target);
+  }
+  addTodo = event => {
+    event.preventDefault();
+    this.setState(
+          {
+           toDos:[...this.state.toDos, {task: this.state.inputText}],
+           inputText :''
+          }
+    );
+  }
   render() {
     return (
       <div>
```

Comment:
Great candidate for a `<Fragment>` here since it's just a wrapping div and you're not using it for anything other than wrapping.